tests/foreman/api/test_contentview.py

```diff
@@ -1080,6 +1080,7 @@ def test_positive_admin_user_actions(
         content_view.read()
 
 
+@pytest.mark.e2e
 @pytest.mark.tier2
 def test_positive_readonly_user_actions(target_sat, function_role, content_view, module_org):
```

Comment:
I don't see how this would be an E2E test for content view. I would imagine a content view E2E test would create, add repos, publish, promote, delete, etc.

Reply:
there's a decent candidate in the UI suite, but I was attempting to find the closest that matches that in API. Would you think that this: https://github.com/SatelliteQE/robottelo/blob/b4b471aac102cfaeb40cffda7cd8452f1a2c1e1f/tests/foreman/ui/test_contentview.py#L100

would be more suitable? It fits most of the CRUD criteria, aside from deletion.

Reply:
Would it be acceptable to mark this as a github issue and assign myself, but merge with the UI version marked as e2e for the time being? We've been asked to try to get e2e markers in wherever we can currently, or close the story as "no candidate" otherwise. Good improvement ideas though.

Reply:
I'd say so. What do @SatelliteQE/robottelo-tier-2-reviewers think?

Reply:
https://github.com/SatelliteQE/robottelo/issues/11312